README.md

```diff
@@ -1,12 +1,15 @@
 # Issue-Finder
-A search engine for newbie-friendly github issues.
+
+> A search engine to find newbie-friendly github issues. The issue finder helps you find a list of beginner-friendly GitHub repos, backed by Y Combinator, that have Ruby and Javascript issues that are good for new contributors.
```

Comment:
Hello @Benmuiruri,

This intro paragraph now explains more effectively what Issue Finder is. 🙌 

Great contribution!